bibliopixel/layout/geometry/matrix.py

```diff
@@ -73,16 +73,13 @@ def make_matrix_coord_map_multi(config, rotation=0, y_flip=False):
     matrix_result = []
 
     def populate_config(cfg):
-        result = copy.copy(DEFAULT_CONFIG)
-        result.update(cfg)
-        return result
+        return dict(DEFAULT_CONFIG, **cfg)
 
     def add_row(offset, matrix, row):
-        count = sum([len(y) for y in row])
         for y in row:
             matrix.append([x + offset for x in y])
-        offset += count
-        return matrix, offset
```

Comment:
why did you remove matrix?

Reply:
You don't need it - it's exactly the same matrix that you pass in.  You just mutate it a bit.  :-D